pyproject.toml

```
@@ -1,3 +1,76 @@
+[build-system]
+requires = ["setuptools>=64"]
```

(Personal preference, non-blocking) 
```suggestion
requires = ["setuptools >= 64"]
```